packages/augur-state/src/db/Rollback.test.ts

```diff
@@ -1,104 +1,31 @@
 import { Augur } from "@augurproject/api";
 import { uploadBlockNumbers } from "@augurproject/artifacts";
 import settings from "@augurproject/state/src/settings.json";
-import { DB, UserSpecificEvent } from "../db/DB";
+import { DB } from "../db/DB";
 import { makeMock } from "../utils/MakeMock";
 import { ContractDependenciesEthers } from "contract-dependencies-ethers";
-import { EthersProvider, Web3AsyncSendable } from "ethers-provider";
+import { makeTestAugur, AccountList, genericEventNames, userSpecificEvents } from "./test";
 
-const TEST_NETWORK_ID = 4;
-
-// TODO Get these from GenericContractInterfaces (and do not include any that are unneeded)
-const genericEventNames: Array<string> = [
-    "DisputeCrowdsourcerCompleted",
-    "DisputeCrowdsourcerCreated",
-    "DisputeWindowCreated",
-    "MarketCreated",
-    "MarketFinalized",
-    "MarketMigrated",
-    "MarketParticipantsDisavowed",
-    "ReportingParticipantDisavowed",
-    "TimestampSet",
-    "TokensBurned",
-    "TokensMinted",
-    "UniverseCreated",
-    "UniverseForked",
-];
 
-// TODO Update numAdditionalTopics/userTopicIndexes once contract events are updated
-const userSpecificEvents: Array<UserSpecificEvent> = [
-  {
-    "name": "CompleteSetsPurchased",
-    "numAdditionalTopics": 3,
-    "userTopicIndex": 2,
-  },
-  {
-    "name": "CompleteSetsSold",
-    "numAdditionalTopics": 3,
-    "userTopicIndex": 2,
-  },
-  {
-    "name": "DisputeCrowdsourcerContribution",
-    "numAdditionalTopics": 3,
-    "userTopicIndex": 1,
-  },
-  {
-    "name": "DisputeCrowdsourcerRedeemed",
-    "numAdditionalTopics": 3,
-    "userTopicIndex": 1,
-  },
-  {
-    "name": "InitialReporterRedeemed",
-    "numAdditionalTopics": 3,
-    "userTopicIndex": 1,
-  },
-  {
-    "name": "InitialReportSubmitted",
-    "numAdditionalTopics": 3,
-    "userTopicIndex": 1,
-  },
-  {
-    "name": "InitialReporterTransferred",
-    "numAdditionalTopics": 2,
-    "userTopicIndex": 2,
-  },
-  {
-    "name": "MarketMailboxTransferred",
-    "numAdditionalTopics": 3,
-    "userTopicIndex": 2,
-  },
-  {
-    "name": "MarketTransferred",
-    "numAdditionalTopics": 2,
-    "userTopicIndex": 1,
-  },
-  {
-    "name": "OrderCanceled",
-    "numAdditionalTopics": 3,
-    "userTopicIndex": 2,
-  },
-  {
-    "name": "OrderCreated",
-    "numAdditionalTopics": 3,
-    "userTopicIndex": 0,
-  },
-  {
-    "name": "OrderFilled",
-    "numAdditionalTopics": 2,
-    "userTopicIndex": 1,
-  },
-  {
-    "name": "TokensTransferred",
-    "numAdditionalTopics": 3,
-    "userTopicIndex": 2,
-  },
+const mock = makeMock();
+const TEST_NETWORK_ID = 4;
+const ACCOUNTS: AccountList = [
```

Comment:
Seems like this should probably moved elsewhere so it can be reused across multiple tests instead of being duplicated.